.github/workflows/main.yml

```diff
@@ -0,0 +1,121 @@
+        restore-keys: ${{ runner.os }}-godot-release_debug_editor
+    
+    # Use python 3.x release (works cross platform; best to keep self contained in it's own step)
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.x' 
+        # Optional - x64 or x86 architecture, defaults to x64
+        architecture: 'x64' 
+
+    # Setup scons, print python version and scons version info, so if anything is broken it won't run the build.
+    - name: Configuring python packages
+      run: |
+        python -c "import sys; print(sys.version)"
+        python -m pip install scons pywin32
+        python --version
+        scons --version
+
+    # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
+    - name: Compilation
+      run: |
+        scons -j2 compdb=no verbose=yes warnings=all werror=yes platform=windows tools=yes target=release_debug
+
+# Build Product Upload (tested and working)
+# sorry this is disabled until github can give us some more space as we would hit our limit very quickly
+# tested this code and it works fine so just enable it to get them back
+#    - name: publishing godot windows-editor
+#      uses: actions/upload-artifact@v1
+#      with:
+#        name: windows-editor (x64)
+#        path: bin/godot.windows.opt.tools.64.exe
+
+
+  windows-template:
+    runs-on: "windows-latest"
+    name: Windows Template (release)
+    steps:
+    - uses: actions/checkout@v2 
+    
+    # Cache must be put in the github workspace directory on windows to be persisted correctly
+    - name: "Force scons cache to the correct value for this builder"
+      run: echo "::set-env name=SCONS_CACHE::${{env.HOME}}/.scons_cache/"
+
+    # Upload cache on completion and check it out now
+    - name: Configure .scons_cache and download cache
+      id: scons-cache
+      uses: RevoluPowered/cache@v2.1
+      with:
+        path: ${{env.SCONS_CACHE}}
+        key: ${{ runner.os }}-godot-release_runtime
+        restore-keys: ${{ runner.os }}-godot-release_runtime
+    
+    # Use python 3.x release (works cross platform)
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.x' 
+        # Optional - x64 or x86 architecture, defaults to x64
+        architecture: 'x64' 
+
+    # You can test your matrix by printing the current Python version
+    - name: Configuring Python packages
+      run: |
+        python -c "import sys; print(sys.version)"
+        python -m pip install scons pywin32
+        python --version
+        scons --version
+    - name: Compilation
+      run: |
+        scons -j2 compdb=no verbose=yes warnings=all werror=yes platform=windows target=release tools=no
```

Comment:
What does `compdb=no` do? We don't have this option currently.

Reply:
sorry meant compiledb=no :)

Reply:
should turn off compilation database

Reply:
removed entirely